docs/devel/running-locally.md

```diff
@@ -68,6 +69,12 @@ You need an [etcd](https://github.com/coreos/etcd/releases) in your path, please
 
 You need [go](https://golang.org/doc/install) in your path (see [here](development.md#go-versions) for supported versions), please make sure it is installed and in your ``$PATH``.
 
+#### OpenSSL
+
+You need [OpenSSL](https://www.openssl.org/) installed.  If you do not have the `openssl` command available, you may see the following error in `/tmp/kube-apiserver.log`:
+
+```server.go:333] Invalid Authentication Config: open /tmp/kube-serviceaccount.key: no such file or directory```
```

Comment:
Hey @mikedanese, I copied it from the line below (with the git command) and it looks fine to me on GitHub. I'm unclear what you mean by "This doesn't look right." - it'd be helpful if you could elaborate a bit. I was trying to be consistent with the rest of the file, but will revise the PR to put the backticks on their own line.